test/sanity/validate-modules/main.py

```diff
@@ -384,6 +384,17 @@ def _check_for_gpl3_header(self):
                 msg='GPLv3 license header not found'
             )
 
+    def _check_for_simple_gpl3_header(self):
+        if self._is_new_module():
+            if len([line for line in self.text.split('\n')
+                    if 'GNU General Public License' in line]) > 1:
+                self.reporter.error(
+                    path=self.object_path,
+                    code=108,
+                    msg='GPLv3 simple license header not found: '
+                        'http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright'
```

Comment:
I wonder if we should direct to the `devel` version of the docs, as this will be for new module development

http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#copyright

Reply:
Also, all links should be `https` instead of `http`.

Reply:
I'm wondering if we should combine the two GPL license checks into a single more comprehensive test, instead of adding a second one. We could check for:

- Correct license
- Short form (for new modules)
- Correct placement near the top of the file

This would be more thorough than the current substring matching which looks for text strings anywhere in the module.

Also, it appears the logic in the original test is faulty:

```python
        if ('GNU General Public License' not in self.text and
                 'version 3' not in self.text):
```

Reply:
@mattclay - it's not apparent to me in what way that logic is faulty.

I've fixed all of your other comments though.